timemachine/cpp/src/neighborlist.cu

```diff
@@ -142,34 +154,50 @@ std::vector<std::vector<int> > Neighborlist<RealType>::get_nblist_host(
 
     gpuErrchk(cudaFree(d_coords));
     gpuErrchk(cudaFree(d_box));
+    gpuErrchk(cudaFree(d_rebuild));
 
     return ixn_list;
 
 }
 
+template<typename RealType>
+void Neighborlist<RealType>::setup_neighborlist(int *d_rebuild_nblist, const bool force, cudaStream_t stream) {
+    // reset the interaction count
+    if (force) {
```

Comment:
hm I don't quite like that there's two ways to trigger a rebuild, one way by passing in a CPU "force" variable, and the other via `*d_build_nblist`.